tests/infra_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	netURL "net/url"
```

Comment:
I think imports don't usually use Capital, like `strconv`

Reply:
just because URL is abbreviation, wdyt?
but added an alias just because i use vars named url, else we could just use the folder name "url" without alias

Reply:
ok, I don't know of an official convention so I'll take it either way.